src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total, perPage, currentPage, onPageChange,
+}) => {
+  const pageCount = Math.ceil(total / perPage);
+  const pages = getNumbers(1, pageCount);
+  const firstPage = currentPage === 1;
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', { disabled: firstPage })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={firstPage}
+          onClick={() => {
+            if (firstPage) {
+              return;
+            }
+
+            onPageChange(currentPage - 1);
+          }}
```

Comment:
make one handler for next and prev buttons.
which can accept two arguments and return arrow function
```
const handlePageChange = (page, preventChange) => () => {
            if (preventChange) {
              return;
            }

            onPageChange(page);
          }
```
```suggestion
          onClick={handlePageChange(currentPage - 1, isFirstPage)}
```